.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 50
```

Comment:
- [ ] You should not update the `yml `file. It is added so that you follow best practice and maintain consistency with other developers. In this case the Microverse students. In the real job you will not have the luxury of modifying configurations.